src/client/tile.cpp

```diff
@@ -1735,6 +1735,43 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 			baseimg->drop();
 			baseimg = image;
 		}
+		/*
+			[opacity:R
+			Makes the base image transparent according to the given ratio.
+			R must be between 0 and 255.
+			0 means totally transparent.
+			255 means totally opaque.
+		*/
+		else if (str_starts_with(part_of_name, "[opacity:"))
+		{
+			if (baseimg == NULL) {
+				errorstream << "generateImagePart(): baseimg == NULL "
+						<< "for part_of_name=\"" << part_of_name
+						<< "\", cancelling." << std::endl;
+				return false;
+			}
+
+			Strfnd sf(part_of_name);
+			sf.next(":");
+			std::string ratio_str = sf.next("");
+
+			int ratio = 0;
+
+			if (is_number(ratio_str))
+				ratio = mystoi(ratio_str, 0, 255);
+
+			core::dimension2d<u32> dim = baseimg->getDimension();
+
+			for (u32 y = 0; y < dim.Height; y++)
+			for (u32 x = 0; x < dim.Width; x++)
+			{
+				video::SColor c = baseimg->getPixel(x,y);
+				if (c.getAlpha() != 0) {
+					c.setAlpha(ratio);
```

Comment:
l tried the qtiplot program, it's the first time l used it, so ple
![2016-07-09-120559_1920x1080_scrot](https://cloud.githubusercontent.com/assets/3192173/16707439/fe3d06c2-45cd-11e6-9a1b-7231c1355e2d.png)
left one is Krock's and right one is floor(BASE_ALPHA * ALPHA_MOD / 255 + 0.5)